datalake/src/test/java/com/sequenceiq/datalake/flow/SdxReactorFlowManagerTest.java

```diff
@@ -102,9 +107,12 @@ public void testSdxBackupOnUpgradeAwsSupported() {
 
     @Test
     public void testSdxBackupOnUpgradeAzureSupported() {
+        DetailedEnvironmentResponse detailedEnvironmentResponse = new DetailedEnvironmentResponse();
+        detailedEnvironmentResponse.setCloudPlatform("MOCK");
```

Comment:
Why does setting to AZURE not work here?

Reply:
My bad, i should have used MOCK here. Fixed it now. Good catch.